index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>| Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="mobile-main">
+        <div class="mobile-header">
+            <div class="app-bar">
+                <div class="logo">
+                    My Logo
+                </div>
+                <div class="menu">
+                    <img src="/Union1.png" alt="">
+                </div>
+            </div>
+            <div class="header-main-content">
+                <div class="introduction-primary-text">
+                    I’m Andile
+                    Glad to see you!
+                </div>
+
+                <div class="introduction-supporting-text">
+                    I’m a software developer! I 
+                    can help you build a product 
+                    , feature or website Look 
+                    through some of my work and 
+                    experience! If you like what 
+                    you see and have a project you 
+                    need coded, don’t hestiate 
+                    to contact me.
+                </div>
+
+                <div class="main-content-contacts-text">
+                    Let's connect
+                </div>
+                
+                <div class="main-content-contacts">
+                    <img src="/social media icons.jpg" alt="">
+                </div>
```

Comment:
This looks fire 🔥 but it's just an image for now, which makes people unable to click each links to reach you on social media 😢 so The list of icons should use a `ul` tag to group them, and `li` per icon; each icon should have a link (`a` tag).
  * Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. 🤔